tests/unit/components/playback/controls/AddCustomer.spec.js

```diff
@@ -120,9 +244,274 @@ describe("AddCustomer.enter_uuidbase57", () => {
       await Vue.nextTick();
       expect(spied_text_validator).toHaveBeenCalledWith(entry);
 
-      expect(target_error_message.text()).toStrictEqual(
-        spied_text_validator.mock.results[0].value
+      expect(target_error_message.text()).toStrictEqual(response_text);
```

Comment:
why did you change this?  We had a specific discussion in Slack about trying to keep the tests in AddCustomer from being brittle by not testing the internal behavior of the TextValidation function
https://curibio.slack.com/archives/DQYMJLLAV/p1604414727006900

Reply:
Completed.